dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeFactory.java

```diff
@@ -260,7 +264,29 @@ public static void saveMultiTrees(List<MultiTree> mTrees) throws DotDataExceptio
 
     }
 
+    @WrapInTransaction
+    public static void saveMultiTrees(String pageId, List<MultiTree> mTrees) throws DotDataException {
+        if (mTrees == null || mTrees.isEmpty())
+            throw new DotDataException("empty list passed in");
+
+        DotConnect db = new DotConnect().setSQL(DELETE_ALL_MULTI_TREE_SQL)
+                .addParam(pageId);
+        db.loadResult();
+
+        final List<Params> params = list();
+        int i = 0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "i" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)